src/main/java/uk/gov/pay/api/utils/TrustStoreLoader.java

```diff
@@ -0,0 +1,64 @@
+package uk.gov.pay.api.utils;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.NoSuchFileException;
+import java.nio.file.Paths;
+import java.security.KeyStore;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.security.cert.Certificate;
+import java.security.cert.CertificateException;
+import java.security.cert.CertificateFactory;
+
+public class TrustStoreLoader {
```

Comment:
This should only load when the JVM starts, isn't this what we want? Certs may change, and a restart should reload new certs.